plugins/builddep.py

```diff
@@ -115,7 +115,10 @@ def run(self):
             rpm.delMacro(macro[0])
 
         if pkg_errors:
-            raise dnf.exceptions.Error(_("Some packages could not be found."))
+            msg = _("Some packages could not be found.")
+            if not self.opts.skip_broken:
+                msg += " " + _("Try to add '--skip-broken' to skip uninstallable packages.")
```

Comment:
I would recommend to add this hint in documentation with warning but keep original statement here. I suggest that this is a workaround that could lead in unpredictable behavior during builds.